Gemfile.lock

```diff
@@ -1,26 +1,25 @@
 PATH
   remote: .
   specs:
-    heartcheck-sidekiq (0.1.1)
+    heartcheck-sidekiq (0.1.2)
```

Comment:
What about removing the Gemfile.lock from git?

For me, this is a [good practice in gem projects](https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/)

Reply:
Nice one! Removed!